components/core/src/clp_s/clp-s.cpp

```diff
@@ -80,6 +81,17 @@ int main(int argc, char const* argv[]) {
             return 1;
         }
 
+        auto timestamp_dict = clp_s::ReaderUtils::read_timestamp_dictionary(archives_dir);
```

Comment:
```suggestion
        auto const timestamp_dict = clp_s::ReaderUtils::read_timestamp_dictionary(archives_dir);
```

Reply:
Since the timestamp dictionary returned here is a shared pointer I don't think adding const here does much. I have now gone through and made all of the accessors for TimestampDictionaryReader const though.

Reply:
I think it’s just the guideline saying that we should add const when it’s possible. I’m 99% sure clang-tidy will also suggest this

Reply:
Double checked and clang-tidy does not suggest const for this case.